### PR TITLE
feat: First setup of example-content

### PR DESCRIPTION
**Pull request by usm-joost**
A first setup of example content.
To view the new content pages locally, you can go to the examples overview page (http://localhost:1313/examples/)

This includes:
- Basic markdown examples
- Block markdown examples
- Custom front-matter explanation
- Shortcode examples

During the writing of the examples, some elements where edited/fixed:
- Fixes in base styling
- Renamed config/element sticky_note > notification
- Renamed shortcode - alert > info / warning / caution
- Fixed/added session-based hiding of notifications

Signed-off-by: Joost Passon <joost.passon@usmedia.nl>

### CODE REVIEW

**Review by myitcv — APPROVED** [review]
LGTM (test)

**Review by myitcv — COMMENTED** [review]
@usm-joost looking more closely, this PR also appears to contain changes unrelated to the adding of example pages. Is this correct?

**Comment by myitcv** [comment]
@Lyoness - please can you review this change too?

File: content/en/examples/_index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                            </a>
+                        </li>
+                        <li class="nav__item">
+                            <a class="nav__link" href="#" disabled>
+                                <span class="nav__text">Text</span>
+                            </a>
+                        </li>
+                        <li class="nav__item">
+                            <a class="nav__link" href="#" disabled>
+                                <span class="nav__text">Logo's</span>
```
Comment (Lyoness) [inline content/en/examples/_index.html:132]: s/Logo's/Logos

File: content/en/examples/basic/front-matter/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Front Matter
+weight: 14
+---
+
+Hugo uses front-matter to add meta information to a contentpage. This can be done various formats - in the CUE website we prefer **YAML** (To be decided).
```
Comment (Lyoness) [inline content/en/examples/basic/front-matter/index.md:6]: nit: add a space.

File: content/en/examples/shortcodes/buttons/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+{{< button href="/" label="Go to the homepage" >}}
+{{< button href="/" label="Go to the homepage" icon="download" >}}
+{{< button href="/" label="Go to the homepage" theme="outline" >}}
+
+## Attributes
+
+href
+: required - Enter the target ie. url of page you want to link to
```
Comment (Lyoness) [inline content/en/examples/shortcodes/buttons/index.md:23]: nit: `target, i.e.,`

**Review by Lyoness — COMMENTED** [review]
I just had small nits, otherwise LGTM 👍🏼

File: content/en/examples/basic/front-matter/index.md (hunk 2 above)
Comment (Lyoness) [reply content/en/examples/basic/front-matter/index.md:6]: s/contentpage/content page

**Comment by usm-joost** [comment]
> @usm-joost looking more closely, this PR also appears to contain changes unrelated to the adding of example pages. Is this correct?

Looking at the changes / files - this looks as expected for me.
The reason more then just 'content/en/examples/...' was changed, is because I also needed to update/fix things that I saw while writing the content.

File: content/en/examples/shortcodes/buttons/index.md (hunk 3 above)
Comment (usm-joost) [reply content/en/examples/shortcodes/buttons/index.md]: Fixed

File: content/en/examples/basic/front-matter/index.md (hunk 2 above)
Comment (usm-joost) [reply content/en/examples/basic/front-matter/index.md]: Fixed

File: content/en/examples/_index.html (hunk 1 above)
Comment (usm-joost) [reply content/en/examples/_index.html]: Fixed

**Comment by myitcv** [comment]
@usm-joost 

> The reason more then just 'content/en/examples/...' was changed, is because I also needed to update/fix things that I saw while writing the content.

Understood. 

We tend to either split out such changes into a separate PR/CL, or say as much in the commit message. Happy either way for this change.

Indeed on the subject of the commit message, please can you ensure the commit message reflects the description in the PR above? 

That way, our git history can be read entirely from `git log`, rather than having to click through to a PR.

Thanks

**Comment by myitcv** [comment]
Thanks - importing via GerritHub now.

**Comment by myitcv** [comment]
Closed via https://github.com/cue-lang/cuelang.org/commit/0125235acc729cfdb9a234e31878a3f4212d5575. 

(It didn't close automatically because of https://github.com/cue-lang/cue/issues/2238)

**Pull request closed without merging**